app/presenters/statistics_sort_presenter.rb

```diff
@@ -0,0 +1,98 @@
+class StatisticsSortPresenter < BaseSortPresenter
```

Comment:
There is a lot of duplicated code in ```SortPresenter```. Can we remove some of this? It might make it clearer what the difference is between ```SortPresenter``` and ```StatisticsSortPresenter```